.eslintrc.json

```
@@ -45,6 +45,7 @@
   },
   // add your custom rules here
   "rules": {
+    "no-console": "off",
```

Don't know that I particularly agree with turning this off altogether. We definitely don't want to allow `log` or `table` calls to remain in the code, but I see the use for having `warn` or `error` calls. Maybe we do something like this?

```suggestion
    "no-console": ["error", { allow: ["warn", "error"] }],
```